articles/openstack/ostack-how-static-ip.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Static IP  | UKCloud Ltd
+description: Instructions on how to create static IPs and assign them to instances
+services: openstack
+author: Bryce Nicholls
+toc_rootlink: How to
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create Static IPs for assignment to instances.
+toc_fullpath: How To/ostack-how-static-ip.md
+toc_mdlink: ostack-how-static-ip.md
+---
+
+# Creating Static IPs for assignment to instances.
+
+Static IPs  are assigned to instances in openstack by creating ports and adding them to the instance.
+
+A port is a connection point for attaching a single device, such as the NIC of a server, to a network. The port also describes the associated network configuration, such as the MAC and IP addresses to be used on that port.
+
+## Steps
+
+> ![NOTE]
```

Comment:
For notes the exclamation mark needs to go inside the square bracket:

\[!NOTE]